docs/logging.md

```diff
@@ -1,58 +1,68 @@
 # Logging
 
-Using Bigflow, you can easily add a handler to your root logger that will send all logging calls to the Cloud Logging.
-In the CL, you can filter logs per project id/logger name and workflow id. 
 
-## Getting started
-To use logging, you have to install `bigflow` with `logger` module - `bigflow[log]`.
-To create a logger with the CL handler, you have to use `configure_logging` method.
+With Bigflow you can easily configure Python logging to send all messages the Cloud Logging (Stackdriver).
+Cloud Logging allows you to filter logs by project id / log name / workflow id and other fields.
```

Comment:
what other fields?

Reply:
Technically StackDriver allows you to filter by any available field.
In our case it might be severity, timestamp, python module/filename (with structured logging) etc.
Not sure that we need to list all of them here - CL provides UI for this (click on the field).

Reply:
Maybe provide some link here to stackdriver docs instead of writing `another fields`

Reply:
I've removed 'other fields' completely from documentation, as it should be small and concise.